ludwig/datasets/configs/mercedes_benz_greener.yaml

```diff
@@ -5,6 +5,7 @@ archive_filenames: mercedes-benz-greener-manufacturing.zip
 sha256:
   mercedes-benz-greener-manufacturing.zip: 91143716085345a84dc4991b8eb1d5ff80d8aa134930de946b3b24be0f2e5d1a
 train_filenames: train.csv
+test_filenames: test.csv
```

Comment:
Likewise, `train.csv` has a "y" column, while test.csv does not.  Perhaps, instead of including these as the "test set", we should add a new category .... `kaggle_test_filenames` perhaps?

Or maybe 'unlabeled_filenames` to be more general and less kaggle-specific?